builtin/providers/aws/resource_aws_directory_service_directory.go

```diff
@@ -89,33 +89,20 @@ func resourceAwsDirectoryServiceDirectory() *schema.Resource {
 			},
 			"type": &schema.Schema{
 				Type:     schema.TypeString,
-				Computed: true,
+				Optional: true,
+				Default:  "SimpleAD",
+				ForceNew: true,
```

Comment:
I think we should consider adding a `ValidateFunc` here in order to provide feedback as early as possible?